### ISSUE TITLE
Harmonics editor features

### ISSUE THREAD

**Issue by suhr** [zynaddsubfx/zyn-fusion-issues#15]
1. Drawing though (so you can draw basic shape with one move)
2. Different scales (logarithmic, for example)

**Comment by fundamental** [comment]
I agree with 1 and 2 is already taken care of via mag. type controls in both the oscillator views and the subsynth harmonic view.

**Comment by suhr** [comment]
> type controls in both the oscillator views and the subsynth harmonic view

Indeed. But dB scale looks something like `V = k₀ exp(v)`. What about `V = k₀ (exp(v) - 1)` scale?

**Comment by unfa** [comment]
I think the Mag Type names do scare poeple off.

Maybe something like "Scale precision" Low, Medium, High, Very High would be an easier thing for artists to use?
Or maybe just "Linear" or "Logarithmic"? (maybe "Logarithmic 2" and "3" also)
 Maybe this feature would be used more if the names wouldn't scare poeple off?

**Comment by fundamental** [comment]
Does the behavior shown in https://youtu.be/fs0Q1y49WMY provide the functionality needed for the 'drawing through' request?

The example video involves dragging across harmonics with left click and clearing by dragging with middle click.
